packages/truffle-db/src/workspace/test/index.ts

```diff
@@ -58,7 +62,14 @@ describe("ContractNames", () => {
     expect(data).toHaveProperty("contractNames");
 
     const { contractNames } = data;
-    expect(contractNames).toEqual([]);
+    const dirExists = await fse.exists(path.join(tempDir.name, ".db", "contract"));
+
+    if(dirExists) {
+     expect(contractNames).toEqual(["MagicSquare", "SquareLib", "Migrations", "VyperStorage", "Migrations"]);
+    } else {
+       expect(contractNames).toEqual([]);
+    }
```

Comment:
I think this should be two separate test cases (one for it existing, one for it not)

Reply:
This should just actually always be blank, because we haven't added contract names when the test is originally run. We add contracts elsewhere, and test for persistence so I think its ok to leave just the test for the empty array (will still be empty even with persistence since I remove the .db folder in test cleanup each time).